static/css/components/check-in.less

```diff
@@ -61,7 +61,7 @@
   margin-top: -15px;
   padding-top: 9px;
   border: 1px solid @light-grey;
-  font-size: .8em;
+  font-size: .7em;
```

Comment:
Let's use the css font size variables; this should be @font-size-label-medium

Reply:
Yes, good idea thank you!